lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1085,64 +1085,89 @@ def test_source_is_empty(self):
     @ddt.data(
         (
             {
-                'desktop_webm': 'https://webm.com/dw.webm',
-                'hls': 'https://hls.com/hls.m3u8',
                 'youtube': 'v0TFmdO4ZP0',
-                'desktop_mp4': 'https://mp4.com/dm.mp4'
+                'hls': 'https://hls.com/hls.m3u8',
+                'desktop_mp4': 'https://mp4.com/dm.mp4',
+                'desktop_webm': 'https://webm.com/dw.webm',
             },
             ['https://www.youtube.com/watch?v=v0TFmdO4ZP0']
         ),
         (
             {
-                'desktop_webm': 'https://webm.com/dw.webm',
-                'hls': 'https://hls.com/hls.m3u8',
                 'youtube': None,
```

Comment:
would you please add a case where YT id is `''`

Reply:
Never mind, please ignore this.

Reply:
I think `youtube: None` checks the same thing which `youtube: ''` will be checking. Both conditions i.e `None` and `''`  are both false.
Is there anything we can discuss? happy to talk about it further.

Reply:
Alright. Thanks.